.drone.yml

```diff
@@ -1,7 +1,6 @@
 # We publishLocal the maven plugin instead of compile to check the maven plugin integration
 # Only test sbt-bloop 1.x because of bug in `^`
 
-# TODO: re-add `nativeBridge/test`
```

Comment:
The CI for JDK 11 is failing because we added back these tests, which is what we should do :smile: Would you mind trying to dig on the reasons why JDK 11 is doesn't work well with our Scala Native tests? Everything seems to work on JDK 8 (ignore Windows).

Reply:
There are some issues around JDK 11, e.g. https://github.com/scala-native/scala-native/issues/1681

Reply:
Since JDK 8 is the official supported JDK by Scala Native I'm skipping the tests for that target.
When the bug is solved I can re-enable that.
Thank you for pointing out @ekrich